pkg/kubectl/cmd/create_service.go

```diff
@@ -51,20 +51,22 @@ var (
 
 	serviceClusterIPExample = templates.Examples(i18n.T(`
     # Create a new ClusterIP service named my-cs
-    kubectl create service clusterip my-cs --tcp=5678:8080
+    kubectl create service clusterip my-cs --ports=tcp:5678:8080
 
     # Create a new ClusterIP service named my-cs (in headless mode)
     kubectl create service clusterip my-cs --clusterip="None"`))
 )
 
 func addPortFlags(cmd *cobra.Command) {
 	cmd.Flags().StringSlice("tcp", []string{}, "Port pairs can be specified as '<port>:<targetPort>'.")
+	cmd.Flags().MarkDeprecated("tcp", "It will be removed in the future, please use --ports instead")
```

Comment:
Was this change discussed and signed of by sig-cli?

cc @kubernetes/sig-cli-pr-reviews

Reply:
I think this is fine.
@fabianofranz Thoughts?